frontend/i18n-scripts/memsource-download.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+
+source ./i18n-scripts/languages.sh
+
+while getopts p: flag
+do
+  case "${flag}" in
+      p) PROJECT_ID=${OPTARG};;
+      *) echo "usage: $0 [-p]" >&2
+      exit 1;;
+  esac
+done
+
+echo "Checking if git workspace is clean"
+GIT_STATUS="$(git status --short --untracked-files -- public/locales packages/**/locales)"
+if [ -n "$GIT_STATUS" ]; then
+  echo "There are uncommitted files in public or package locales folders. Remove or commit the files, then run this script again."
+  git diff
+  exit 1
+fi
+
+echo "Downloading PO files from Project ID \"$PROJECT_ID\""
+
+DOWNLOAD_PATH="$(mktemp -d)" || { echo "Failed to create temp folder"; exit 1; }
+
+# Memsource job listing is limited to 50 jobs per page
+# We need to pull all the files down by page and stop when we reach a page with no data
+for i in "${LANGUAGES[@]}"
+do
+  COUNTER=0
+  CURRENT_PAGE=$(memsource job list --project-id "$PROJECT_ID" --target-lang "$i" -f value --page-number 0 -c uid | tr '\n' ' ')
+  until [ -z "$CURRENT_PAGE" ]
+  do
+    ((COUNTER++))
+    echo Downloading page "$COUNTER"
+    memsource job download --project-id "$PROJECT_ID" --output-dir "$DOWNLOAD_PATH"/"$i" --job-id "$CURRENT_PAGE"
+    CURRENT_PAGE=$(memsource job list --project-id "$PROJECT_ID" --target-lang "$i" -f value --page-number "$COUNTER" -c uid | tr '\n' ' ')
+  done
+done
+
+# Memsource gives us back file names like "console-app_package%3Dconsole-app."
+# We want names like "console-app_package=console-app" (with an equals sign)
+# in order to use our existing name-based file sorting mechanism.
```

Comment:
Honestly, I think it would be better to just avoid the special characters in the filename altogether, particular if Memsource does weird things with them. Then we can also avoid needing the `rename` utility. Would a convention like `my-package__my-namespace` be simpler? (Similar to BEM names.)

Reply:
I can make the change, but I'd have to rework two other scripts.

Reply:
> I can make the change, but I'd have to rework two other scripts.

I think it's worth doing.